src/nbl/video/CVulkanImageView.cpp

```diff
@@ -0,0 +1,45 @@
+#include "nbl/video/CVulkanImageView.h"
+
+#include "nbl/video/CVKLogicalDevice.h"
+#include "nbl/video/CVulkanImage.h"
+
+namespace nbl {
+namespace video
+{
+
+CVulkanImageView::CVulkanImageView(CVKLogicalDevice* _vkdev, SCreationParams&& _params) : IGPUImageView(std::move(_params)), m_vkdev(_vkdev)
+{
+    auto* vk = m_vkdev->getFunctionTable();
+    auto vkdev = m_vkdev->getInternalObject();
+
+    VkImageViewCreateInfo ci;
+    ci.sType = VK_STRUCTURE_TYPE_IMAGE_VIEW_CREATE_INFO;
+    ci.pNext = nullptr;
```

Comment:
are there any extensions that can be chained?

Reply:
I repeat my question

Reply:
yes, https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkImageViewCreateInfo.html basically ctrl+f "pnext"

Reply:
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkSamplerYcbcrConversionInfo.html and https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkImageViewUsageCreateInfo.html
are the only options

Reply:
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkImageFormatListCreateInfo.html

also this

Reply:
usage flags looks like a nice thing to support

Reply:
@achalpandeyy look at this, then click "resolve"

Reply:
@achalpandeyy did you see the conversation?

Reply:
yep

resolving..